src/applications/disability-benefits/all-claims/config/8940/index.js

```diff
@@ -63,6 +65,13 @@ export default function() {
         schema: hospitalizationHistory.schema,
       },
       // 8940 - Doctor Treatment
+      unemployabilityDoctorCare: {
+        title: "Doctor's care",
```

Comment:
This should use an apostrophe (`’`) instead of a `'` mark. There are several instances of that in this PR, can you go through and update them?